source/optional_features/delayed_capture/index.html.md.erb

```diff
@@ -37,3 +37,32 @@ payment flow. Otherwise, it will expire.
 If your service has enabled payment receipt emails, GOV.UK Pay will send a
 payment receipt email to the user once your service's capture request is
 received and accepted.
+
+## See the capture URL for a payment
+
+You can see the capture URL for a single payment in GOV.UK Pay API responses.
+
+For example, take the payment `nj5hlp736as5ohelmkrg35ippr`.  
+
+If the payment is available for capture, the capture `property` will have the 
+status `AWAITING CAPTURE REQUEST`. This would appear in:  
```

Comment:
This isn’t correct: the `"capture"` URL appears in the `"_links"` object (as detailed below) but there are no other changes